.github/PULL_REQUEST_TEMPLATE/deep_connector.md

```diff
@@ -0,0 +1,15 @@
+# Installation 
+MailMonkey installation image:
+![image](https://github.com/user-attachments/assets/ebc027fb-b82b-4505-ac71-f2b61209fa4d)
+
+# Read
+Insert image of console.
```

Comment:
```suggestion
- [] Screenshot of operations on dashboard
```
We've started preferring the term 'dashboard' rather than console, so a quick suggestion here :)